### DISCUSSION TITLE
[OS] 프로세스와 스레드

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/challenge-every-week-posting#56] [3. CS(컴퓨터 공학) 관련 글 모음]
컴공생들한테는 상식이지만 우테코 로드맵에 있어서 정리해봤습니다.
전공 수업에서 배웠던 기억이 새록새록 나네요^-^
[게시글 링크](https://amaran-th.github.io/Java/[OS]%20%ED%94%84%EB%A1%9C%EC%84%B8%EC%8A%A4%EC%99%80%20%EC%8A%A4%EB%A0%88%EB%93%9C/)

**Comment by shkisme** [comment]
흐어어어어어
운체 기말고사 어떡하죠? 종강을 달라ㅏㅏㅏㅏㅏㅏ

...ㅎㅎ 글 잘읽었습니다 😇
